src/Dashboard/Dashboard.tsx

```diff
@@ -31,7 +31,7 @@ export const Dashboard = () => {
   const navigateTo = useNavigateTo()
 
   const wallet = useSelectedWallet()
-  const {isLoading: isFetchingUtxos, refetch: refetchUtxos} = useUtxos(wallet)
+  const {isLoading: isSync, sync} = useSync(wallet)
```

Comment:
sync triggers utxos sync

Reply:
What do you think about `syncing` or `isSyncing`?